pytest No module named eth_utils

* Version: 1.0.0
* Python: 2.7.14-2ubuntu1 / 3.6.3-0ubuntu2
* OS: ubuntu 17.10 artful

### What was wrong?

compiling with either python3 or python, it doesn't seem to matter: eth_utils is not found by pytest.

make test
py.test tests
========================================================================== test session starts ===========================================================================
platform linux2 -- Python 2.7.14+, pytest-3.2.1, py-1.4.34, pluggy-0.4.0 -- /usr/bin/python
cachedir: .cache
rootdir: /home/z/python/eth-utils/github-eth-utils/eth-utils, inifile: pytest.ini
plugins: hypothesis-3.12.0
collected 0 items / 11 errors                                                                                                                                             

================================================================================= ERRORS =================================================================================
___________________________________________________________ ERROR collecting tests/abi-utils/test_abi_utils.py ___________________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/abi-utils/test_abi_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/abi-utils/test_abi_utils.py:3: in <module>
    from eth_utils.hexidecimal import (
E   ImportError: No module named eth_utils.hexidecimal
_______________________________________________________ ERROR collecting tests/address-utils/test_address_utils.py _______________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/address-utils/test_address_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/address-utils/test_address_utils.py:5: in <module>
    from eth_utils.address import (
E   ImportError: No module named eth_utils.address
______________________________________________________ ERROR collecting tests/currency-utils/test_currency_tools.py ______________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/currency-utils/test_currency_tools.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/currency-utils/test_currency_tools.py:9: in <module>
    from eth_utils.currency import (
E   ImportError: No module named eth_utils.currency
____________________________________________________ ERROR collecting tests/encoding-utils/test_big_endian_integer.py ____________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/encoding-utils/test_big_endian_integer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/encoding-utils/test_big_endian_integer.py:10: in <module>
    from eth_utils.encoding import (
E   ImportError: No module named eth_utils.encoding
_________________________________________________ ERROR collecting tests/formatting-utils/test_padding_left_and_right.py _________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/formatting-utils/test_padding_left_and_right.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/formatting-utils/test_padding_left_and_right.py:5: in <module>
    from eth_utils.formatting import (
E   ImportError: No module named eth_utils.formatting
________________________________________________ ERROR collecting tests/functional-utils/test_return_value_decorators.py _________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/functional-utils/test_return_value_decorators.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/functional-utils/test_return_value_decorators.py:3: in <module>
    from eth_utils import (
E   ImportError: No module named eth_utils
____________________________________________ ERROR collecting tests/hexidecimal-utils/test_0x_prefix_addition_and_removal.py _____________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/hexidecimal-utils/test_0x_prefix_addition_and_removal.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/hexidecimal-utils/test_0x_prefix_addition_and_removal.py:5: in <module>
    from eth_utils.hexidecimal import (
E   ImportError: No module named eth_utils.hexidecimal
____________________________________________________ ERROR collecting tests/hexidecimal-utils/test_is_0x_prefixed.py _____________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/hexidecimal-utils/test_is_0x_prefixed.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/hexidecimal-utils/test_is_0x_prefixed.py:5: in <module>
    from eth_utils.hexidecimal import (
E   ImportError: No module named eth_utils.hexidecimal
________________________________________________________ ERROR collecting tests/hexidecimal-utils/test_is_hex.py _________________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/hexidecimal-utils/test_is_hex.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/hexidecimal-utils/test_is_hex.py:5: in <module>
    from eth_utils import (
E   ImportError: No module named eth_utils
______________________________________________________ ERROR collecting tests/string-utils/test_string_coersions.py ______________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/string-utils/test_string_coersions.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/string-utils/test_string_coersions.py:10: in <module>
    from eth_utils.string import (
E   ImportError: No module named eth_utils.string
_________________________________________________________ ERROR collecting tests/types-utils/test_types_utils.py _________________________________________________________
ImportError while importing test module '/home/z/python/eth-utils/github-eth-utils/eth-utils/tests/types-utils/test_types_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/types-utils/test_types_utils.py:3: in <module>
    from eth_utils.types import (
E   ImportError: No module named eth_utils.types
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 11 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================== 11 error in 0.56 seconds ========================================================================
Makefile:28: recipe for target 'test' failed
make: *** [test] Error 2


### How can it be fixed?

??

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,8 @@
+0.7.2
+-----
+
+* Minor fix for how `__version__` is computed in the `eth_utils` module.
+
 0.7.1
 -----
 

--- a/README.md
+++ b/README.md
@@ -1,8 +1,8 @@
 # Ethereum Utils
 
-[![Join the chat at https://gitter.im/ethereum/eth_utils](https://badges.gitter.im/ethereum/eth-utils.svg)](https://gitter.im/ethereum/eth-utils)
+[![Join the chat at https://gitter.im/ethereum/eth-utils](https://badges.gitter.im/ethereum/eth-utils.svg)](https://gitter.im/ethereum/eth-utils)
 
-[![Build Status](https://travis-ci.org/ethereum/eth_utils.png)](https://travis-ci.org/ethereum/eth-utils)
+[![Build Status](https://travis-ci.org/ethereum/eth-utils.png)](https://travis-ci.org/ethereum/eth-utils)
 
 
 Common utility functions for codebases which interact with ethereum.

--- a/eth_utils/__init__.py
+++ b/eth_utils/__init__.py
@@ -80,4 +80,4 @@ from .types import (  # noqa: F401
 )
 
 
-__version__ = pkg_resources.get_distribution("eth_utils").version
+__version__ = pkg_resources.get_distribution("eth-utils").version

